indico/modules/designer/__init__.py

```diff
@@ -80,13 +80,20 @@ class PageLayout(RichIntEnum):
 
 
 @signals.core.get_placeholders.connect_via('designer-fields')
-def _get_notification_placeholders(sender, **kwargs):
+def _get_notification_placeholders(sender, regform=None, **kwargs):
```

Comment:
Hmm, why is the method named this way? It has nothing to do with notifications :thinking: